core-web/libs/template-builder/src/lib/components/template-builder/utils/mocks.ts

```diff
@@ -1,5 +1,8 @@
+import * as _ from 'lodash';
```

Comment:
we have plans to remove `lodash` from the project. Please don't use it

Reply:
Here is the issue to do that [Remove Lodash from the core-web](https://github.com/dotCMS/core/issues/24842)